spec/features/posts_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] I believe we are testing the `feature` type here, not `model`. Kindly fix that.